api/src/main/java/javax/enterprise/inject/spi/ProcessBeanAttributes.java

```diff
@@ -92,4 +92,18 @@
      * @throws IllegalStateException if called outside of the observer method invocation
      */
     public void veto();
+
+
+    /**
+     * Instructs the container to ignore all non-static, final methods with public, protected or default visibility declared on
+     * any bean type of the specific bean during validation of injection points that require proxyable bean type.
+     * These methods shouldn't be invoked on the bean instances.
+     *
+     * Calling this method will bypass standard rules for unproxyable bean types (section 3.11 of the spec)
```

Comment:
I am fine with any wording as long as we keep the "warning sign" up so we avoid people misusing/overusing this.